files/zh-cn/learn/javascript/objects/object-oriented_programming/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: 面向对象编程基本概念
+slug: Learn/JavaScript/Objects/Object-oriented_programming
+tags:
+  - JavaScript
+  - Learn
```

Comment:
Hi @tgz033. We just do not need this in translated document. See: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE